apache/zeppelin
# [ZEPPELIN-3198] UI should not show Version/GIT Control if the same if not supported

### What is this PR for?
Currently, UI shows an option for version/GIT Control even when it is not supported by the underlying implementing storage configuration.

It is only after users try to save a commit and get an error "Couldn't checkpoint note revision: possibly storage doesn't support versioning. Please check the logs for more details.".

So, if implementing storage configuration doesn't support git storage, UI should not show those options.

### What type of PR is it?
[Improvement]

### What is the Jira issue?
* [ZEPPELIN-3198](https://issues.apache.org/jira/projects/ZEPPELIN/issues/ZEPPELIN-3198)

### How should this be tested?
On using "org.apache.zeppelin.notebook.repo.GitNotebookRepo"  for `zeppelin.notebook.storage`  user should see revision/version control option, for rest of the others e.g. "FileSystemNotebookRepo" user should not see that option 


### Questions:
* Does the licenses files need update? N/A
* Is there breaking changes for older versions? N/A
* Does this needs documentation? N/A
